django_mongodb/utils.py

```diff
@@ -25,6 +26,39 @@ def check_django_compatability():
         )
 
 
+def parse(uri):
+    uri = parse_uri(uri)
+    port = None
+
+    if uri["fqdn"] is None:
+        # If fqdn is None then this is not a SRV URI, so we need to extract the
```

Comment:
Personal preference: write comments without "we".   "so extra..." saves a few words.